exampleSite/resources/_gen/assets/scss/scss/coder-dark.scss_9e20ccd2d8034c8e0fd83b11fb6e2bd5.content

```diff
@@ -0,0 +1,126 @@
+body.colorscheme-dark {
```

Comment:
You've added these scss files by mistake. Please, remove the `resources` folder from inside `exampleSite`.
Also, there is a simple conflict to solve in `CONTRIBUTORS.md`.